### DISCUSSION TITLE
繼續努力深淵💪

### DISCUSSION THREAD

**Discussion by Andrea0126** [share121/inter-knot#4991] [原神]
![Screenshot_2024-09-16-22-02-47-652_com miHoYo GenshinImpact](https://github.com/user-attachments/assets/43465740-a83e-4a04-be6b-d6a054f62929)
之前12層都打不過，這次至少都拿到2星，比以前多賺100原石☺️
開心(⁠ ⁠´⁠◡⁠‿⁠ゝ⁠◡⁠`⁠)

**Comment by passbyDFX** [comment]
hh，现在都没玩了，现在就光玩穿越和我的世界了
